runtime/shared_common/Managers.cpp

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2001, 2014 IBM Corp. and others
+ * Copyright (c) 2001, 2020 IBM Corp. and others
```

Comment:
Nothing is changed in this file, so no need to change the copyright date.

Reply:
Fixed